pkg/fiat/client.go

```diff
@@ -0,0 +1,85 @@
+package fiat
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+//go:generate counterfeiter . Client
+type Client interface {
+	Authorize(account string) (Response, error)
+}
+
+func NewClient(url string) Client {
+	return &client{
+		url: url,
+	}
+}
+
+func NewDefaultClient() Client {
+	return &client{
```

Comment:
Change the return for the default client to `return NewClient("http://spin-fiat.spinnaker:7003")`. Maybe set the default URL as a const if you want.